tutu-frontend/src/app/hakemus/[oid]/perustelu/yleiset/perustelut/components/SelvitysTutkinnonAsemasta.tsx

```diff
@@ -0,0 +1,18 @@
+import { useHakemus } from '@/src/context/HakemusContext';
+import { useTranslations } from '@/src/lib/localization/hooks/useTranslations';
+
+import { Muistio } from '@/src/components/Muistio';
+
+export const SelvitysTutkinnonAsemasta = () => {
```

Comment:
Tämän ja perustelut-yleiset--selvitys-tutkinnon-myontajasta -muistion voisi mun mielestä ympätä ihan tuonne page.tsx suoraan ellei näitä käytetä muualla? Meillä on kuitenkin jo toi muistio-komponentti